### ISSUE TITLE
Example source

### ISSUE THREAD

**Issue by Costava** [PhilippHochmann/ctable#4]
The README should mention that the code to generate the tables in the example images can be seen in `tests/test_table.c`

Even better would be showing example usage code in the README.md or a standalone example file.

**Comment by PHochmann** [comment]
I agree this is necessary, since the functions and especially order of function calls needed is not very intuitive.

**Comment by PHochmann** [comment]
See https://github.com/PhilippHochmann/ctable/commit/f215abfda7483aeead693c7048b031d956858006 for function documentation. May be extended by example code snippets and explanations.
